src/tests/fixtures/comments.fixture.js

```diff
@@ -12,3 +17,25 @@ export const badRequest = {
 export const noTripFound = {
   comment: faker.lorem.sentence()
 };
+
+export const deleteComment = {
```

Comment:
Can you rename this to another name. `deleteComment` sounds like a function name